packages/mongodb-rag-ingest/src/commands/pages.ts

```diff
@@ -48,6 +55,12 @@ const commandModule: CommandModule<
 
 export default commandModule;
 
+export const doInitPagesCommand = async ({ pageStore }: ResolvedConfig) => {
+  if (pageStore) {
+    await pageStore?.init?.();
+  }
```

Comment:
Should we throw an error if the page store isn't defined?

Reply:
i think we should